website/docs/deploy/configuring-unleash-v3.md

```diff
@@ -40,10 +40,11 @@ unleash.start(unleashOptions);
   - _database_ - the database name to be used (`DATABASE_NAME`)
   - _ssl_ - an object describing ssl options, see https://node-postgres.com/features/ssl (`DATABASE_SSL`, as a stringified json object)
   - _version_ - the postgres database version. Used to connect a non-standard database. Defaults to `undefined`, which let the underlying adapter to detect the version automatically. (`DATABASE_VERSION`)
-  - _pool_ - an object describing pool options, see https://knexjs.org/#Installation-pooling. We support the following three fields:
+  - _pool_ - an object describing pool options, see https://knexjs.org/guide/#pool. We support the following four fields:
     - _min_ - minimum connections in connections pool (defaults to 0) (`DATABASE_POOL_MIN`)
     - _max_ - maximum connections in connections pool (defaults to 4) (`DATABASE_POOL_MAX`)
     - _idleTimeoutMillis_ - time in milliseconds a connection must be idle before being marked as a candidate for eviction (defaults to 30000) (`DATABASE_POOL_IDLE_TIMEOUT_MS`)
+    - \_afterCreate - a callback for for configuring active connections, see https://knexjs.org/guide/#aftercreate. This is incompatible with the `ALLOW_NON_STANDARD_DB_DATES` environment variable, which will override this property to support non standard postgres date formats. If you've set your postgres instance to use a date style other than `ISO, DMY` then you'll need to set the `ALLOW_NON_STANDARD_DB_DATES` environment variable to `true`. Setting the environment variable should be preferred over writing your own callback
```

Comment:
This is great! 😄 Just a few small suggestions:

```suggestion
    - _afterCreate_ - a callback for for configuring active connections, see https://knexjs.org/guide/#aftercreate. This is incompatible with the `ALLOW_NON_STANDARD_DB_DATES` environment variable, which will override this property to support non-standard Postgres date formats. If you've set your Postgres instance to use a date style other than `ISO, DMY` then you'll need to set the `ALLOW_NON_STANDARD_DB_DATES` environment variable to `true`. Setting the environment variable should be preferred over writing your own callback.
```

I'm assuming this option does not have an env var equivalent (seeing as it's a function)

Reply:
You are correct sir, there's no equivalent env var for this, the env var mentioned here will override this property to inject the handling function